### DISCUSSION TITLE
How to inventory Yealink T33G IP Phones

### DISCUSSION THREAD

**Discussion by Etarcos01** [glpi-project/glpi-agent#766] [Q&A]
IP PHONE INVENTORY WITH GLPI AGENT 
Hello everyone,
I need your help.
I need to do an inventory of my IP phones (YEALINK T33G) with the GLPI 1.9 agent.
My phones don't support SNMP or SSH.

How can I proceed? Thank you in advance for your help.

**Comment by g-bougard** [comment]
Hi @Etarcos01 

sorry, there's no dedicated method to inventory such proprietary devices.

Then I guess you can only inventory them manually in GLPI.
